### ISSUE TITLE
Application Type - Custom Dropdown Menu 

### ISSUE THREAD

**Issue by samsadasivan** [cds-snc/platform-forms-client#5837]
**_Specs: Windows
Screen Readers Used: JAWS, NVDA
Browsers Tested: Google Chrome, Microsoft Edge, Internet Explorer_**

This issue observed on 2 screens 

**Components:**  
“Application Type” and “Years of experience” 

**Observation:** 
1. Lack of Instruction to Expand Dropdown: The screen reader doesn't indicate that the dropdown can be expanded or how. 

2. No Guidance on Navigation and Selection: Screen reader users receive no feedback or guidance when navigating options. 

3. No Confirmation of Selection: Once an item is selected, screen reader users are not notified. 

**_Compounded Severity: Critical_**

**Recommendation:**  
1. Dropdown Expansion Instruction: 

Ensure the dropdown control is an accessible button or combobox element with ARIA roles and appropriate instructions. 

Provide textual instruction (e.g., add a visually hidden instruction like “Press Enter or Space to expand the options”). 

2. Selection Guidance: 

Provide keyboard support (Arrow Down/Up, Enter to select). 

Include accessible instructions for navigation (e.g., via an aria-describedby attribute pointing to a hidden label like “Use arrow keys to navigate”). 

3. Selection Confirmation: 

Programmatically update the dropdown label with the selected option and announce it using live region. 

Alternatively, update the aria-labelledby or ensure the selected item text is visually and programmatically conveyed

**Comment by thiessenp-cds** [comment]
I disagree with this recommendation. 
If the dropdown were a custom widget, then yes the advice about adding aria roles, states, and arguable some instructions would make sense. But the dropdown is a native html select option. (see attached)
It's always best to use native html when possible for better screen reader support. 
I'll close this ticket.


<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNXkrQ2c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--de57a11bf66db37adbcf8c4e234b5077e0496897/Screenshot%202025-07-16%20at%201.54.13%E2%80%AFPM.png" alt="Screenshot 2025-07-16 at 1.54.13 PM.png" />
